### PR TITLE
sdk/Jenkinsfile: port to Docker

### PR DESCRIPTION
**Pull request by pookimon**
Vagrant to Docker migration for SDK Jenkins job.
All mentions of vagrant dir and vagrant ssh were cleared
and changed to sh

Signed-off-by: Dmytro Iurchuk <diurchuk@luxoft.com>

### CODE REVIEW

**Comment by pookimon** [comment]
Build failed due unrelated to change issues

**Comment by sashko** [comment]
> Build failed due unrelated to change issues

Restart it ;)

File: sdk/Jenkinsfile [hunk 1]
```diff
@@ -1,63 +1,41 @@
...
-        }
+    stage("Copying latest SDK ${variant}") {
+        sh "rm -rf ${archive_directory}"
 
+        copyArtifacts(projectName: "${upstream}",
+                      selector: lastSuccessful(),
+                      filter: "artifacts_${variant}/${sdkFilePattern}")
+    }
+    def customImage = docker.image("pelux/pelux-yocto:yoctouser")
+    customImage.inside('-v $WORKSPACE:/workspace -v /var/yocto-cache:/var/yocto-cache --cap-add=NET_ADMIN --device=/dev/net/tun') {
```
Comment (sashko) [inline sdk/Jenkinsfile]: can't we get it from Jenkins job parameter?

**Comment by pookimon** [comment]
It's not working this way:)

-- The CXX compiler identification is unknown
CMake Error at CMakeLists.txt:9 (project):
  The CMAKE_CXX_COMPILER:

    /home/yoctouser/sdk/sysroots/x86_64-peluxsdk-linux/usr/bin/x86_64-pelux-linux/x86_64-pelux-linux-g++

  is not a full path to an existing compiler tool.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.

File: sdk/Jenkinsfile (hunk 1 above)
Comment (pookimon) [reply sdk/Jenkinsfile]: What exactly? Image name?
Multibranch jobs are not parametrized. We can add it as a parameter but in the script (and"pelux/pelux-yocto:yoctouser" will be default value)

**Comment by sashko** [comment]
Do we even use this?

**Comment by pookimon** [comment]
> Do we even use this?

SDK test job was kicked off in April last time, but as I understand every part of project should be ported to docker anyway.

File: sdk/Jenkinsfile [hunk 2]
```diff
@@ -1,63 +1,41 @@
...
         stage("Installing SDK ${variant}") {
-            vagrant("/vagrant/sdk/install_sdk ${sdkDir} /vagrant/${archive_directory}")
+            sh "/workspace/sdk/install_sdk ${sdkDir} /workspace/${archive_directory}"
         }
-
+
```
Comment (sashko) [inline sdk/Jenkinsfile]: remove whitespaces

**Pull request merged**